docs/virtual_machines/net_binding_plugins/macvtap.md

```diff
@@ -0,0 +1,227 @@
+# Macvtap binding
+
+## Overview
+
+With the `macvtap` binding plugin, virtual machines are directly exposed to the Kubernetes
+nodes L2 network. This is achieved by 'extending' an existing network interface
+with a virtual device that has its own MAC address.
+
+Its main benefits are:
+
+- Direct connection to the node nic with no intermediate bridges.
+
+### Functionality support
+| Functionality                                  | Support |
+|------------------------------------------------|---------|
+| Migration support                              | No      |
+| IPAM support (on pod)                          | No      |
+| Require extra capabilities (on pod) to operate | No      |
```

Comment:
The "supports - no" give the feeling it is a disadvantage. I would change to "run without extra capabilities (on pod) to operate - support yes".

Reply:
Done